Esterni/VerbaliEsterni/2020_12_31/components/verbale.tex

```diff
@@ -0,0 +1,89 @@
+\section{Informazioni generali}
+
+\begin{itemize}
+
+	\item \textbf{Canale di comunicazione:} Zoom;
+	
+	\item \textbf{Data:} \DataMeeting{};
+	
+	\item \textbf{Ora inizio:} 10:30;
+	
+	\item \textbf{Ora fine:} 11:00;
+	
+	\item \textbf{Segretario:} \Tommaso{};
+	
+	\item \textbf{Partecipanti}:  e alcuni componenti del gruppo \Gruppo{}:
+	
+		\begin{itemize}
+			\item \Daniele{};
+			\item \Davide{};
+			\item \Francesco{};
+			\item \Giosue{};
+			\item \Lucrezia{};
+			\item \Tommaso{}.
+			
+		\end{itemize}
+
+	\item \textbf{Membri assenti}:
+		\begin{itemize}
+			\item \Matteo{}.
+		\end{itemize}
+	\end{itemize}
+\section{Ordine del giorno}
+
+\begin{itemize}
+	\item\textbf{Discussione sul corretto utilizzo del glossario;}
+	\item\textbf{Discussione sullo snellimento dei singoli documenti;}
+	\item\textbf{Discussione sulle informazioni ridondanti nei vari documenti.}
+
+
+\end{itemize}
+
+\newpage
+
+
+\section{Resoconto}
+La riunione è stata richiesta dal gruppo \textit{\Gruppo{}} per poter chiarire dubbi sorti durante la stesura della documentazione.
+Durante l'incontro è stato possibile sottoporre una serie di quesiti di seguito riportati.
```

Comment:
sostituire "sottoporre" con "discutere di "
sostituire " di seguito riportati " con "riportati in seguito."